assets/ideogram-style.css

```diff
@@ -1,47 +1,163 @@
-.ideogram-banner {
-    height: 75px;
-    background-color: rgb(66, 196, 247);
-    border-radius: 2px;
+#ideogram-body {
+    font-family: sans-serif;
+    font-weight: 200;
+    padding: 5px;
+    background-color: #fdfcff;
+    color: #303030;
+    height: calc(100vh - 70px);
+    overflow: auto;
+}
+#ideogram-body ::-webkit-scrollbar {
+    width: 4px;
+    height: 0px;
+    margin-right: 500px;
+    background-color: #ffdfcff;
+}
+#ideogram-body ::-webkit-scrollbar-thumb {
+    width: 2px;
+    background-color: #230047;
+    border-radius:15px;
+    padding-right:5px;
+}
+
+#ideogram-wrapper::-webkit-scrollbar {
+    display: none;
+}
+
+#ideogram-control-tabs {
+    width: 400px;
+    height: 550px;
+    margin: 50px;
+    margin-right: 0px;
+    background-color: #fff;
+    font-size: 10pt;
+    float: left;
+}
+
+#ideogram-control-tabs .tab-content {
+    padding: 20px;
+
+    height: 400px;
+    margin: 20px;
+    overflow-y: auto;
+}
+#ideogram-control-tabs .tab--selected {
+    border-top: solid 2px #230047;
+    color: #230047;
+}
+.ideogram-tab {
+    height: 400px;
+    overflow-y: auto;
+    padding-right: 15px;
 }
 
-.ideogram-banner h2{
-    color: white;
-    margin-left: 2%;
+.ideogram-controls-name {
     display: inline-block;
+    margin-top: 15px;
+    font-weight: 400;
+    vertical-align: middle;
+    width: 160px;
+    font-size: 14pt;
+    margin-bottom: 20px;
 }
 
-.ideogram-banner Img{
-    position: relative;
+#ideogram-feature-dropdown {
+    display: inline-block;
     float: right;
-    right: 10px;
-    height: 75px;
+}
+#ideogram-container {
+    display: inline-block;
+    width: 800px;
+    height: 550px;
+    overflow: auto;
+    margin-left: 50px;
+    margin-top: 50px;
+    padding-right: 20px;
 }
 
-.ideogram-column {
-    padding-left: 0.5%;
-    max-height: 83vh;
-    overflow-y: auto;
+#_ideogramMiddleWrap {
+    height: 550px;
 }
 
-.ideogram-custom {
-    max-height: 83vh;
-    overflow-y: auto;
+#ideogram-body .Select {
+    display: inline-block;
+    width: 140px;
     position: relative;
+    top: 10px;
 }
 
-.ideogram-column-content {
+#ideogram-body #chr-select-1 .Select, #ideogram-body #chr-select-2 .Select, #ideogram-body #chr-brush .Select {
+    width: 65px !important;
+}
+#ideogram-body #chr-select-1 .Select {
+    margin-right: 5px;
+    margin-left: 0px;
+}
+#ideogram-body #chr-select-2 .Select {
+    margin-left: 5px;
+    margin-right: 0px;
+}
+.Select .ideogram-dropdown {
+    display: inline-block;
+}
+
+input.ideogram-homology-inputs, input.ideogram-annot-inputs {
+    display: inline-block;
+    width: 140px;
+}
+
+#ideogram-body hr {
     width: 100%;
 }
+#organism-change, #orientation-switch, #bandlabel-switch, #chromlabel-switch, #rotatable-switch, #sex-switch, #fullband-switch, #resolution-select, #chr-brush, #annotation-select, #orientation-anote {
```

Comment:
I don't know how to break a line in CSS; is it possible?

Reply:
Yep! Fixed in bfcb660